docs/en_US/install/3ds.md

```diff
@@ -28,8 +39,46 @@ We make use of 3 Luma patches to connect your console to Pretendo:
 2) `0004013000003202` - Friends system module. This patch replaces the `https://nasc.nintendowifi.net` URL with our servers URL 
 3) `0004013000003802` - act system module. This patch replaces the `https://account.nintendo.net/v1/api/` URL with our servers URL
 
-## How does it work?
+<div class="tip">
+ℹ️ <b>Skip this step if Luma patches are already enabled on your console.</b>
+</div>
+
+In order to make use of the patches listed above, you will need to enable Luma patches on your console. Hold the SELECT button on your 3DS and power it on.
+
+You should be greeted to a black screen with a few tickboxes. You'll need to tick 'enable loading external FIRMS and modules' and 'enable game patching' to ensure Pretendo works correctly. After that, press START on your console to save and boot into your 3DS.
+
+## Installing Nimbus app to HOME Menu
+
+<div class="tip">
+ℹ️ <b>Skip this step if you downloaded the 3dsx only zip file.</b>
+</div>
+
+If you downloaded the combined or cia zip files, you can install the Nimbus application to the HOME Menu for quick and easy access. Here's how to do that.
+
+First of all, open up the FBI application on your 3DS HOME Menu. Once it is loaded, click on SD and then click on cias. Next, locate `nimbus.cia` and click on it. In the menu that appears, press the install and delete CIA button. This installs the application to your HOME Menu.
+
+Once it has finished installing, press the HOME button and exit FBI. You should see a message that a new application has been added to the HOME Menu. Click OK and you'll now have Nimbus on your HOME Menu for easy access.
+
+## Using Nimbus
+
+To use Pretendo servers instead on Nintendo's servers, you'll need to open up Nimbus. If you followed the previous step, you can find it on your HOME Menu, or you can also find it in the Homebrew Launcher. Once inside Nimbus, click on Pretendo to switch to your Pretendo profile.
+
+You can also repeat the same steps to go back to the Nintendo profile if you want to use Nintendo's servers.
+
+Your active profile persists between reboots, so there's no need to open up Nimbus to switch to Pretendo (or to Nintendo) every time you turn on your 3DS.
```

Comment:
It's already clear based on earlier information what Nimbus does, a lot of this can be simplified

```suggestion
## Using Nimbus

Depending on how you installed Nimbus, launch it either through the Homebrew Launcher or the 3DS Home MENU. Select either `Pretendo` or `Nintendo` to swap between services

Your selection persists between reboots
```